<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JumpHeightTestParamsDialog</class>
 <widget class="QDialog" name="JumpHeightTestParamsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Параметры теста высоты прыжков</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QLabel" name="lblFinishKind">
        <property name="text">
         <string>Завершение теста</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbFinishKind"/>
      </item>
      <item>
       <widget class="QFrame" name="frJumpsCount">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="lblJumpsCount">
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Количество прыжков</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="edJumpsCount">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>200</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frTestTime">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="lblTestTime">
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Время, сек</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="edTestTime">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="minimum">
            <number>5</number>
           </property>
           <property name="maximum">
            <number>300</number>
           </property>
           <property name="singleStep">
            <number>5</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lblStrategy">
        <property name="text">
         <string>Стратегия</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbStrategy"/>
      </item>
      <item>
       <widget class="QFrame" name="frContactTimeBound">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="lblContactTimeBound">
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Временной порог, мсек</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="edContactTimeBound">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="minimum">
            <number>50</number>
           </property>
           <property name="maximum">
            <number>1500</number>
           </property>
           <property name="singleStep">
            <number>50</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QPushButton" name="btnDefault_2">
        <property name="text">
         <string>Сброс к настройкам по умолчанию...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_12">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnOK_2">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel_2">
        <property name="text">
         <string>Отмена</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cbFinishKind</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>JumpHeightTestParamsDialog</receiver>
   <slot>on_finishKindChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbStrategy</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>JumpHeightTestParamsDialog</receiver>
   <slot>on_strategyChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOK_2</sender>
   <signal>clicked()</signal>
   <receiver>JumpHeightTestParamsDialog</receiver>
   <slot>on_ok()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>428</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>556</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel_2</sender>
   <signal>clicked()</signal>
   <receiver>JumpHeightTestParamsDialog</receiver>
   <slot>on_cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>497</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>550</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDefault_2</sender>
   <signal>clicked()</signal>
   <receiver>JumpHeightTestParamsDialog</receiver>
   <slot>on_default()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>552</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_finishKindChanged(int)</slot>
  <slot>on_strategyChanged(int)</slot>
  <slot>on_ok()</slot>
  <slot>on_cancel()</slot>
  <slot>on_default()</slot>
 </slots>
</ui>
